File: src/main/java/kookmin/software/capstone2023/timebank/infrastructure/client/FeignConfiguration.java
package kookmin.software.capstone2023.timebank.infrastructure.client;

import feign.Logger;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Profile;

public class FeignConfiguration {
    @Bean
    @Profile("dev")
    public Logger.Level feignLoggerLevel() {
        return Logger.Level.FULL;
    }
}

File: src/main/java/kookmin/software/capstone2023/timebank/infrastructure/client/FeignConfiguration.kt
package kookmin.software.capstone2023.timebank.infrastructure.client

import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Profile

class FeignConfiguration {
    @Bean
    @Profile("dev")
    fun feignLoggerLevel(): feign.Logger.Level {
        return feign.Logger.Level.FULL
    }
}
